<?xml version="1.0"?>
<launch>
 <!-- ros_args.launch -->
<arg name="filename"/>
<arg name="output_files" default="/home/daniel/metadata"/>
<node pkg="graph_map" type="show_map" name="show_map" args="--file-name $(arg filename) --output-filepath $(arg output_files)" output="screen">
</node>
<node pkg="rviz" type="rviz" name="rviz" args="-d $(find graph_map)/rviz/showMap.rviz" />
</launch>


